README.md

```diff
@@ -21,7 +21,8 @@ This repository contains a curated list of awesome open source libraries that wi
 | [💸 Data Stream Processing](#data-stream-processing) | [:red_circle: Outlier & Anomaly Detection](#outlier-and-anomaly-detection) | [🎁 Feature Store](#feature-store) |
 | [⚔ Adversarial Robustness](#adversarial-robustness) | [💾 Data Storage Optimization](#data-storage-optimisation) | [📓 Data Science Notebook](#data-science-notebook) |
 | [🔥 Neural Search](#neural-search) | [👁️ Industry-strength Computer Vision](#industry-strength-cv) | [🔠 Industry-strength Natural Language Processing](#industry-strength-nlp) |
-| [🍕 Industry-strength Reinforcement Learning](#industry-strength-rl) | [📊 Industry-strength Visualisation](#industry-strength-visualisation) | [💰 Commercial Platform](#commercial-platform) |
+| [🍕 Industry-strength Reinforcement Learning](#industry-strength-rl) | [📊 Industry-strength Visualisation](#industry-strength-visualisation) | [🙌 Industry-strength Recommendation System](#industry-strength-recsys) |
```

Comment:
Here it would be "Recommender System" as opposed to recommendation

Reply:
> Here it would be "Recommender System" as opposed to recommendation

Good question! I have been wandering about the name for a while and decide to use recommendation sys rather than Recommender Sys for the following reasons:
1. https://trends.google.com/trends/explore?q=Recommender%20System,Recommendation%20System "recommendation sys" seems to be used more regularly than "recommender sys"
2. I check the wiki and found they are used interchangeably. It seems like Europe used the "recommender" more compared with North America.
![image](https://user-images.githubusercontent.com/8592144/221968423-7d3be210-0a93-4fbc-9612-f9aa69e59826.png)

Reply:
Thank you for having a look @zhimin-z - that's a good link, let's go for Recommender Systems as it's the more common term in industry

Reply:
> Thank you for having a look @zhimin-z - that's a good link, let's go for Recommender Systems as it's the more common term in industry

Thanks for your feedback @axsaucedo . I am just wondering if we can consider the section name and their usage frequency according to google trends. If we want the list to become more impactful, it is better to use the most frequent names in the domain. Any idea?
![image](https://user-images.githubusercontent.com/8592144/222984186-f23d07a5-d51c-46a1-b394-a15122f34ffa.png)

Reply:
Let's change to "recommender systems", it's the more common term as mentioned.

In regards to metrics google search

6m

![image](https://user-images.githubusercontent.com/1447507/223227361-6b761f14-8cdb-45c0-ae31-020906fbd1d9.png)

vs 1m

![image](https://user-images.githubusercontent.com/1447507/223227335-02d0e95a-90d7-4949-a4de-9c7f618bcd48.png)

Reply:
@axsaucedo @zhimin-z If we compare search results for "system" vs "systems" we get reverse result: 
1.6m
![image](https://user-images.githubusercontent.com/12928778/223249142-c7bf6daf-5e74-44ba-b6a7-0e8957240ba9.png)
vs 9.8m
![image](https://user-images.githubusercontent.com/12928778/223249444-9b628744-028e-4b24-a9a9-3f907a4f3c71.png)

The difference is reflection of user intent. You can look at this [post ](https://www.searchenginejournal.com/singular-plural-keywords/267177/) for details. 

Also note the section is called  that we have is "Industry-strength Recommendation System" where "System" is singular (it matches with other sections).

Reply:
Interesting, @al-yakubovich I wonder if we have the ultimate standard such as using Google trends since sooner or later we would encounter similar naming issues for using a new terminology @axsaucedo

Reply:
@zhimin-z For SEO optimization I would recommend to go with singular terms when possible.  For this case, "recommendation system" is more popular than "recommender system".

Reply:
We're going too many back and forths here , let's please just update to "recommender systems" and merge